docs/application-connector/05-02-application-registry.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Application Registry sub-chart
+type: Configuration
+---
+
+To configure the Application Registry sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **deployment.args.requestTimeout** | Specifies a period of time after which a request to the Application Registry fails to be sent. It is provided in seconds. | `10` |
```

Comment:
The setting allow to specify what is the summary threshold for request handling time. I think the term "time-out" is more suitable.